src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -631,18 +627,20 @@ std::pair<bool, Uptane::Target> SotaUptaneClient::downloadImage(const Uptane::Ta
   return {success, target};
 }
 
-bool SotaUptaneClient::uptaneIteration() {
+bool SotaUptaneClient::uptaneIteration(std::vector<Uptane::Target> *targets, unsigned int *ecus_count) {
```

Comment:
Should we allowed the `targets` parameter to be null? It could help in niche cases, like tests and should be pretty cheap.

Reply:
Done.